charts/cassandra/templates/deployment.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: cassandra
+spec:
+  selector:
+    app.kubernetes.io/app: cassandra
+  ports:
+    - name: jmx
+      protocol: TCP
+      port: 7199
+      targetPort: 7199
+    - name: cassandra
+      protocol: TCP
+      port: 7000
+      targetPort: 7000
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: cassandra
+spec:
+  serviceName: cassandra
+  replicas: 3
+  selector:
+    matchLabels:
+      app.kubernetes.io/app: cassandra
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/app: cassandra
+    spec:
+      containers:
+      - name: cassandra
+        image: {{ .Values.image }}
+        env:
+          - name: CASSANDRA_SEEDS
+            value: "cassandra-0.cassandra.default.svc.cluster.local"
+          - name: JVM_OPTS
+            value: "-Xms1024M -Xmx2048M -XX:MaxPermSize=2048M -XX:MaxHeapSize=2048M"
```

Comment:
I would set min and max equal, and I think 1024 would be ideal to allow for any filesystem caching that might be needed on top of heap allocation.

Reply:
@algchoo I think we still need this

Reply:
To clarify, I think heap min and max can be 1GB, with cgroup limit / request set to 2gb. This allows for any filesystem caching / memory requirements required outside of the JVM heap.